docker-host/main.go

```diff
@@ -0,0 +1,140 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"os/exec"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/filters"
+	"github.com/docker/docker/client"
+	"github.com/robfig/cron/v3"
+	machineryvars "github.com/uselagoon/machinery/utils/variables"
+)
+
+const EnvOverrideHost = "DOCKER_HOST"
+
+var dockerHost = machineryvars.GetEnv("DOCKER_HOST", "docker-host")
+var repo = machineryvars.GetEnv("REPOSITORY_TO_UPDATE", "amazeeio")
```

Comment:
This is an interesting one, it might need to be `uselagoon` though? @tobybellwood ?

Reply:
yah - should be uselagoon here as a default - we use `amazeeio|lagoon` in the charts https://github.com/uselagoon/lagoon-charts/blob/main/charts/lagoon-remote/values.yaml#LL36C23-L36C38 but uselagoon is fine as a default - as long as it still accepts the egrep format needed to use |*() etc